### PR TITLE
Testing delete method

### PR DESCRIPTION
**Pull request by Daniel2894**

### CODE REVIEW

**Comment by Daniel2894** [comment]
Este es el de los gists.

File: test/GithubApi.Gist.test.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        }
+      }
+    };
+
+    let gist;
+    let code;
+    let newGistQuery;
+
+
+    before(() => {
```
Comment (sorregom) [inline test/GithubApi.Gist.test.js]: remove the brackets and the return and it will work the same

File: test/GithubApi.Gist.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      expect(code).to.equal(statusCode.CREATED);
+      expect(gist.description).to.equal(createGist.description);
+      expect(gist.public).to.equal(true);
+      expect(gist).to.containSubset(createGist);
+    });
+
+    describe('Testing if the gist really exist', () => {
+      let gistQuery;
+
+      before(() => {
```
Comment (sorregom) [inline test/GithubApi.Gist.test.js]: you can remove the brackets

File: test/GithubApi.Gist.test.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      it('Should exist the gist created', () => {
+        gistQuery.then((response) => {
+          expect(response.status).to.equal(statusCode.OK);
+        });
+      });
+
+      describe('Deleting the gist', () => {
+        let deleteQuery;
+
+        before(() => {
```
Comment (sorregom) [inline test/GithubApi.Gist.test.js]: you can remove the brackets

File: test/GithubApi.Gist.test.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+        it('Should delete the gist created', () => {
+          deleteQuery.then((response) => {
+            expect(response.status).to.equal(statusCode.NO_CONTENT);
+          });
+        });
+
+        describe('Want to verify if the gist is really gone', () => {
+          let gistQueryDeleted;
+          before(() => {
```
Comment (sorregom) [inline test/GithubApi.Gist.test.js]: you can remove the brackets

**Review by sorregom — COMMENTED** [review]
Please check comments, also the [WIP] tells me to not check this PR

**Review by sorregom — APPROVED** [review]
LGTM, good job @Daniel2894 👍

**Pull request merged**